src/Build/Resources/Strings.resx

```diff
@@ -1751,6 +1751,9 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
   <data name="ErrorWritingCacheFile" UESanitized="false" Visibility="Public">
     <value>MSB4258: Writing output result cache file in path "{0}" encountered an error: {1}</value>
   </data>
+  <data name="InvalidSdkResolverDocumentNode" UESanitized="false" Visibility="Public">
+    <value>MSB4259: Unable to find node {0} in the SDK Resolver XML document</value>
```

Comment:
Something's wrong in our infrastructure; this should have forced you to build and let automation generate to-be-translated strings for our non-English languages. Looking at exactly what's wrong.

Reply:
This CI false-success should be fixed by #4209.  Nothing you did wrong! I'll go ahead and push the xlf files that are changing as a result of your PR.